variables.tf

```
@@ -334,4 +334,10 @@ variable "client_certificate_enabled" {
   description = "Should client certificate be enabled for this Web App?"
   type        = bool
   default     = false
+}
+
+variable "zip_deploy_file" {
+  description = "The quick deployment for webapp using local package."
```

Clarify description:
```suggestion
  description = "The quick deployment for webapp using local package."
```

```suggestion
  description = "The local path of a ZIP-packaged application to deploy to this Web App."
```